app/views/components/input_tele.scala.html

```diff
@@ -49,11 +49,8 @@
         id="@{field.id}"
         name="@{field.name}"
         value="@{field.value}"
-        @if(hint.nonEmpty) {
-            aria-describedby="@{field.id}_hint"
-        }
-        @if(field.hasErrors) {
-            aria-labelledby="error-message-@{field.id}-input"
+        @if(hint.nonEmpty || field.hasErrors) {
+            aria-describedby="@{if(hint.nonEmpty){s"${field.id}_hint"}} @{if(field.hasErrors){s"error-message-${field.id}-input"}}"
```

Comment:
We have had problems with spaces appearing when only one element is present. I think we have markup somewhere to only include the space if there is both.